templates/web-application/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "@workleap/typescript-configs/web-application",
+    "compilerOptions": {
+        "baseUrl": ".",
+        "paths": {
+            "@root/*": ["src/*"]
+        }
```

Comment:
VS https://github.com/patricklafrance/pnpm-ts5-swc-esm-jest/blob/master/tsconfig.json , I removed what's in `@workleap/typescript-configs/web-application`.

Reply:
I would remove the paths (and baseUrl)